e2e_local.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -euo pipefail
+
```

Comment:
Some sort of help output when run without args (or with `-h` or `--help`) would be nice

Reply:
Will do